src/raftstore/store/debug.rs

```diff
@@ -219,6 +220,49 @@ impl Debugger {
         compact_range(db, handle, start, end, false);
         Ok(())
     }
+
+    /// Set a region to tombstone by manual, and apply other status(such as
+    /// peers, version, and key range) from `region` which comes from PD normaly.
+    pub fn set_region_tombstone(&self, id: u64, region: Region) -> Result<()> {
+        let store_id = self.get_store_id()?;
+        if region
+            .get_peers()
+            .iter()
+            .any(|peer| peer.get_store_id() == store_id)
+        {
+            return Err(box_err!("The peer is still in new peers list"));
+        }
+
+        let db = &self.engines.kv_engine;
+        let key = keys::region_state_key(id);
+        match box_try!(db.get_msg_cf::<RegionLocalState>(CF_RAFT, &key)) {
+            Some(mut region_state) => {
+                let conf_ver = region.get_region_epoch().get_conf_ver();
+                if conf_ver == region_state.get_region().get_region_epoch().get_conf_ver() {
+                    return Err(box_err!("The conf_ver hasn't changed."));
```

Comment:
can we meet this case?

Reply:
`remove-peer` in pd-ctl is asynchonrous so that we may run `tikv-ctl tombstone` command before the config really change. So I think add this check is better.

Reply:
If this is happening then it can't pass the check at L231.